src/mbgl/map/transform.cpp

```diff
@@ -517,7 +527,7 @@ double Transform::getAngle() const {
 #pragma mark - Pitch
 
 void Transform::setPitch(double pitch, const Duration& duration) {
-    setPitch(pitch, {NAN, NAN}, duration);
+    setPitch(pitch, ScreenCoordinate {}, duration);
```

Comment:
I originally made the change to rely on NaN for anchors because anchoring at the origin (or somewhere on the top or left side) may be a valid use case. Though it may be difficult to trigger such a transition via a gesture, it may still be possible to do so in the iOS SDK with some combination of edge insets, user location or course tracking, and zooming or rotation gestures. Moreover, it’s only a matter of time before we need to allow arbitrary anchor points for programmatic animations in the mobile SDKs.